docs/disaster-recovery/README.md

```diff
@@ -0,0 +1,29 @@
+# Disaster Recovery
+
+This is a list of guidance for setting up disaster recovery for Cloud Foundry service instances. This guidance is suitable for you if:
+
+**Scenario 1**
+
+- You have two Cloud Foundry cluster in different Azure region. 
+- You want to recover from an outage of data center where your service instance locates
+- You want to recover from an outage of Azure region where your working Cloud Foundry cluster locates
+
+or:
+
+**Scenario 2**
+
+- You have one Cloud Foundry cluster
+- You want to recover from an outage of data center where your service instance locates
+
+
+
+If you have set up disaster recovery successfully, you can:
+
+- For both scenario 1 and scenario 2, recover from an outage of data center where your service instance locates. In this case, if you haven't set up disaster recovery, you can't use the service instance until the data center become available, which may cause your application cannot work during this time. With disaster recovery enabled, you can follow the guidance to start up the backup service instance, make your application use the backup instance, and therefore your application can continue to work ignoring the outage of the data center.
```

Comment:
Didn't feel like smooth reading to me.
* The precondition of this paragraph is `if … set ... successfully, you can`. But the paragraph still talks about `not set`.
* The `become available` doesn't deliver the situation "back to life". Why not just say "recover from a disaster"?
* IMHO, the readability is not good with long sentences... And each of them are made up with many short sentences...
* Why not to use service broker terminology `bind` instead of  `make your application use the backup instance`?

Similar points for the paragraph below.

Reply:
Will polish the paragraph